### PR TITLE
Add showcase section to hackathons

### PR DESCRIPTION
**Pull request by zoek1**
##### Refers/Fixes
https://github.com/gitcoinco/web/issues/6769

##### Testing
image
image
image

### CODE REVIEW

**Comment by PixelantDesign** [comment]
What do you think @fluffays @willsputra

**Comment by fluffays** [comment]
Looks good 🚀 ! I've got 3 small suggestions:

1. **Hacking spotlights**" is a section to showcase cool bounties from the hackathon. Under the description of each bounty, I would also add the winning hackers (where applicable). Ideal format would be:
**Winners:**
Prize #1 - <hacker profile>
Prize #2 - <hacker profile>
etc.

2. "**Top <hackathon_name> Hackers**" => I think it's a bit redundant to include the hackathon name in the title here (since this section is displayed in the hackathon's navigation. I suggest changing this section's title to "Hackers' Wall of Fame".

3. "**Top <hackathon_name> Hackers**" => could we try a vertical alignment of hacker_avatar + hacker_name, instead of putting both on a horizontal line? Something like this:
Screenshot 2020-06-29 at 16 37 51
This might also help with layout on mobile.

**Comment by PixelantDesign** [comment]
What do you think @vs77bb @connoroday since y'all will be using this page?

**Comment by owocki** [comment]
llooks pretty exciting. hoping to get this in for SFBW

File: app/assets/v2/css/gitcoin.css [hunk 1]
```diff
@@ -71,6 +71,7 @@ h4,
   background-position: 0px 0px;
   background-repeat: no-repeat;
   position: relative;
+  z-index: 3;
```
Comment (octavioamu) [inline app/assets/v2/css/gitcoin.css:74]: why is this needed?

Comment (octavioamu) [reply app/assets/v2/css/gitcoin.css:74]: this is used over the site in some places the next box need to overlap the header, for example on onboard.

Comment (zoek1) [reply app/assets/v2/css/gitcoin.css]: good catch, this change was from another PR isn't required here.

**Comment by PixelantDesign** [comment]
Excited for this one!

**Comment by PixelantDesign** [comment]
@frankchen07 need 1 more review!

File: app/dashboard/router.py [hunk 2]
```diff
@@ -59,6 +60,14 @@ class Meta:
...
     """Handle serializing the hackathon object."""
     sponsor_profiles = ProfileSerializer(many=True)
+    prizes = serializers.SerializerMethodField()
+    winners = serializers.SerializerMethodField()
+
+    def get_prizes(self, obj):
+        return Bounty.objects.filter(event=obj).distinct().count()
+
+    def get_winners(self, obj):
+        return reduce(lambda total, prize: total + len(prize.paid), Bounty.objects.filter(event=obj).distinct(), 0)
```
Comment (octavioamu) [inline app/dashboard/router.py:70]: seems even with one winner project still getting 0
